src/azure-cli/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -203,9 +204,14 @@ def _load_certificate_as_bytes(file_name):
 def certificate_type(string):
     """ Loads file and outputs contents as base64 encoded string. """
     import os
-    with open(os.path.expanduser(string), 'rb') as f:
-        cert_data = f.read()
-    return cert_data
+    try:
+        with open(os.path.expanduser(string), 'rb') as f:
+            cert_data = f.read()
+        return cert_data
+    except (IOError, OSError) as e:  # FileNotFoundError introduced in Python 3
+        if e.errno == errno.ENOENT:
+            raise CLIError("Certificate file '{}' does not exist.".format(string))
+        raise CLIError("Unable to load certificate file '{}'.".format(string))
```

Comment:
I suggest include the error detail

Reply:
Changed to use exception's own error message